packages/jsdoc-plugins/lib/relation-fixer/getmissingdocletsdata.js

```diff
@@ -34,14 +34,14 @@ function getMissingDocletsData( originalDoclets, childDoclet, options ) {
 		clonedDoclet.memberof = childDoclet.longname;
 
 		// Add property `inherited` or `mixed`.
-		const relationProperty = getRelationProperty( originalDoclets, childDoclet, d, options.relation );
+		const relationProperty = getRelationProperty( docletCollection.getAll(), childDoclet, d, options.relation );
```

Comment:
Can we pass the whole collection here?

Reply:
We can, but why? does it matter? (maybe I'm missing something)